Lib/test/test_tools/test_i18n.py

```diff
@@ -332,14 +332,14 @@ def test_calls_in_fstring_with_multiple_args(self):
         msgids = self.extract_docstrings_from_str(dedent('''\
         f"{_('foo', 'bar')}"
         '''))
-        self.assertNotIn('foo', msgids)
+        self.assertIn('foo', msgids)
```

Comment:
Both `xgettext` and `pybabel` extract this and it would take a lot of effort to disallow this in general with the current extractor, so I'd leave this for now at least.